index.js

```diff
@@ -0,0 +1,82 @@
+let button = document.querySelector("button")
+let input = document.querySelector("input")
+let inputData = document.querySelector("#inputData")
+let mainData = document.querySelector("#mainData")
+
+
+// button.addEventListener("click", () => {
+//     let p = document.createElement("p")
+//     p.innerHTML = ""
+//     axios.get(`http://localhost:3000/cookBook/`).then(res => {
+//       cookBook.forEach(el => {
+//           let p2 = document.createElement("p2");
+//       })
+//     })
+//   })
+  
+  document.querySelector('#btn1').addEventListener('click', () => {
+    axios.get(`http://localhost:3000/recipes/3`).then(res => {
```

Comment:
You should only have one event listener that listens to different values on a select or input, this way you dont need extra buttons